packages/DatePicker/src/DatePicker.js

```diff
@@ -231,6 +233,7 @@ function DatePicker(props) {
         inputRef={inputRef}
         value={confirmationResult || inputtedString}
         {...extendedInputProps}
+        hasError={hasError}
```

Comment:
@allison-c I'm wondering if it would be better to render the paprika input component in the DatePicker.Input component instead of it just being a shell component? Perhaps something to consider in the future